client/styles/components/_forms.scss

```diff
@@ -27,11 +27,16 @@
   margin-top: #{16 / $base-font-size}rem;
   font-size: #{12 / $base-font-size}rem;
   @include themify() {
-    color: getThemifyVariable('form-navigation-options-color');
+    color: getThemifyVariable("form-navigation-options-color");
+  }
+
+  @media (max-width: 550px) {
+    text-align: center;
+    margin: #{12 / $base-font-size}rem;
```

Comment:
It seems like all this media query does is increase the gap between the "sign up" and "forgot your password" lines?

Reply:
yes